src/List/Title/Title.component.tsx

```diff
@@ -14,7 +14,10 @@ export interface TitleProps extends React.HTMLAttributes<HTMLDivElement> {
 
 const StyledTitle = styled('div')`
     box-sizing: border-box;
-    padding: 0.75rem 1rem;
+    display: flex;
+    padding: 0 1rem 0;
```

Comment:
`0 1rem`?

Reply:
I'm using flex to center the text, but I still need the 16px r/l padding for it to align with the other Items.